### PR TITLE
Added first set of models

### PR DESCRIPTION
**Pull request by Cluttered04**
Added Models for Department, Employee, Computer, and ComputerEmployee

### CODE REVIEW

**Comment by sydneywait** [comment]
⚡️

**Comment by cfitzgerald9** [comment]
🥇 🏋️‍♀️

File: BangazonAPI/Models/Employee.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Employee
+    {
+        public string FirstName { get; set; }
```
Comment (jordan-castelloe) [inline BangazonAPI/Models/Employee.cs:11]: Employees should have a PK

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]
- Add a PK to the Employee model
- In general, we need to have some kind of testing instructions. Even if it's just "the project should build without errors"

**Comment by Cluttered04** [comment]
Fixed missing primary key on Employee model. Program should build without errors.

**Review by jordan-castelloe — APPROVED** [review]

**Pull request merged**